rootfs/api/models/build.py

```diff
@@ -43,10 +43,11 @@ def create(self, user, *args, **kwargs):
         try:
             self.app.deploy(new_release)
             return new_release
-        except Exception:
+        except Exception as e:
             if 'new_release' in locals():
                 new_release.delete()
-            raise
+
+            raise DeisException(str(e)) from e
```

Comment:
We would not, see the from e portion - also you'd get the error message and proper status code up in the view, which is all it would end up getting piped up into and displayed to the end user as